brightcove/brightcove-popup/src/components/gridLayout.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import Loader from "./loader";
+import { Layout } from "../model/layout.model";
+
+const GridLayout: React.FC<Layout> = props => {
+    const { isSelected, selectedVideoList, videos, totalVideos, checkFiles, handleSelect, loadContent } = props;
+    const renderVideos = isSelected ? selectedVideoList : videos;
+
+    return (
+        <ul className="grid-layout">
+            <div className="grid-body">
+                {renderVideos.length > 0 ? (
+                    <>
+                        <ul>
+                            {renderVideos?.map((video) => {
+                                const checked = selectedVideoList.some(
+                                    (check) => check.id === video.id
+                                );
+                                return (
+                                    <li
+                                        title={video.name}
+                                        id={video.id}
+                                        key={video.id}
+                                        className={checked ? "active" : ""}
+                                        onClick={(event) => {
+                                            const liElement = event.currentTarget;
+                                            !checked && video.id === liElement.id
+                                                ? liElement.classList.add("active")
+                                                : liElement.classList.remove("active");
+                                            handleSelect(video);
+                                            const checkbox =
+                                                (liElement.childNodes[0].childNodes[0].childNodes[0] as HTMLInputElement);
```

Comment:
Do code formatting - use prettier on file save
Set `formatOnSave` in vscode settings

Reply:
https://www.robinwieruch.de/how-to-use-prettier-vscode